patching/src/main/java/org/jboss/as/patching/IoUtils.java

```diff
@@ -111,9 +111,14 @@ public static void copyFile(File sourceFile, File targetFile) throws IOException
                     throw PatchLogger.ROOT_LOGGER.cannotCreateDirectory(parent.getAbsolutePath());
                 }
             }
-            final InputStream is = new FileInputStream(sourceFile);
-            final OutputStream os = new FileOutputStream(targetFile);
-            copyStreamAndClose(is, os);
+            try {
+                final InputStream is = new FileInputStream(sourceFile);
+                final OutputStream os = new FileOutputStream(targetFile);
+                copyStreamAndClose(is, os);
```

Comment:
If this code is beeing changed, than it can be changed to
Files.copy(sourceFile.toPath(), targetFile.toPath());

without need for creating extra streams, also error report from this method is already better.